src/containers/SearchPageContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import {
+  getOffers,
+} from "../actions/SearchPageActions";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import SearchPage from "../components/SearchPage/SearchPage";
+
+const initialState = {
+  phrase: "",
+  option: "",
+  filter: "stars",
+  page: 1
+};
+
+export class SearchPageContainer extends Component {
```

Comment:
Here is good example, you split business logic from rendering correctly, not sure if everything works, since I couldn't get filters to work (I guess it's WIP?)